src/Features/Hud/VelocityGraph.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "Hud.hpp"
+#include "Variable.hpp"
+
+class VelocityGraph : public Hud {
+public:
+	VelocityGraph();
+	bool ShouldDraw() override;
+	void GatherData(int slot);
+	void Paint(int slot) override;
+	bool GetCurrentSize(int &xSize, int &ySize) override;
+};
+
+extern VelocityGraph velocityGraph;
+
+extern Variable sar_hud_velocitygraph;
+extern Variable sar_hud_velocitygraph_size;
+extern Variable sar_hud_velocitygraph_font_index;
+
+struct velocity_data_t {
+	int speed;
+	bool on_ground;
+};
+
+inline std::vector<velocity_data_t> data;
```

Comment:
This is broken, and might even cause link errors if imported in multiple files. This should be a static vector within the cpp file, which will also allow moving the struct definition there.